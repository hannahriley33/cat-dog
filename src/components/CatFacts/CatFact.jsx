import React from 'react';
import PropTypes from 'prop-types';

const CatFact = ({ fact }) => (
  <p>
    <span>{fact}</span>
  </p>
);

CatFact.propTypes = {
  fact: PropTypes.array.isRequired
};

export default CatFact;
