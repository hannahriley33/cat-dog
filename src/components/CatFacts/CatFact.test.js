import React from 'react';
import { shallow } from 'enzyme';
import CatFact from './CatFact';

describe('CatFact component', () => {
  it('matches snapshot', () => {
    const fact = [
      { text: 'this is a fact', _id: '12345' } 
    ];
    const wrapper = shallow(<CatFact fact={fact} />);
    expect(wrapper).toMatchSnapshot();
  });
});
