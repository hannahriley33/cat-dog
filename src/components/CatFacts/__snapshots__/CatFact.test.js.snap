// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CatFact component matches snapshot 1`] = `
<p>
  <span>
    <Component />
  </span>
</p>
`;
